src/qtAliceVisionImageIO/QtAliceVisionImageIOHandler.cpp

```diff
@@ -0,0 +1,261 @@
+#include "QtAliceVisionImageIOHandler.hpp"
+
+#include "../utils/jetColorMap.hpp"
+
+#include <QImage>
+#include <QIODevice>
+#include <QFileDevice>
+#include <QVariant>
+#include <QDataStream>
+#include <QDebug>
+
+#include <OpenImageIO/imageio.h>
+#include <OpenImageIO/imagebuf.h>
+#include <OpenImageIO/imagebufalgo.h>
+
+#include <aliceVision/image/io.hpp>
+#include <aliceVision/image/Image.hpp>
+#include <aliceVision/image/pixelTypes.hpp>
+
+#include <iostream>
+#include <memory>
+
+
+inline const float& clamp( const float& v, const float& lo, const float& hi )
+{
+    assert( !(hi < lo) );
+    return (v < lo) ? lo : (hi < v) ? hi : v;
+}
+
+inline unsigned short floatToUShort(float v)
+{
+    return clamp(v, 0.0f, 1.0f) * 65535;
+}
+
+QtAliceVisionImageIOHandler::QtAliceVisionImageIOHandler()
+{
+    qDebug() << "[QtAliceVisionImageIO] QtAliceVisionImageIOHandler";
+}
+
+QtAliceVisionImageIOHandler::~QtAliceVisionImageIOHandler()
+{
+}
+
+bool QtAliceVisionImageIOHandler::canRead() const
+{
+    if(canRead(device()))
+    {
+        setFormat("AliceVisionImageIO");
+        return true;
+    }
+    return false;
+}
+
+bool QtAliceVisionImageIOHandler::canRead(QIODevice *device)
+{
+    QFileDevice* d = dynamic_cast<QFileDevice*>(device);
```

Comment:
You don't check if device can be read ? This function should only return true if openimageio can read this file